### DISCUSSION TITLE
疑似被盗搬至greasyfork

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3277] [综合讨论 / General]
链接：https://greasyfork.org/zh-CN/scripts/443059-bilibili-evolved-preview
看了一下，疑似盗搬🤔

**Comment by Geeyun-JY3** [comment]
锁定用户：https://greasyfork.org/users/2493

目测该用户目前上传的 4 个脚本均为未授权搬运。
（[Super_preloaderPlus_one_改](https://greasyfork.org/scripts/443058) 去年因为 Gressy Fork 政策变动不再符合要求而被[下架](https://github.com/machsix/Super-preloader/issues/646)，另外两个直接完整复制名字就能在 Greasy Fork 找到从名字到简介都一模一样的脚本，都不能叫搬运，离谱）

https://github.com/the1812/Bilibili-Evolved/blob/master/CHANGELOG.md#你可能想问的其他问题
（<https://github.com/the1812/Bilibili-Evolved/blob/539cb50b3614ac6e6b70542b2a799674968960e3/CHANGELOG.md#你可能想问的其他问题>）

已举报：https://greasyfork.org/reports/26522

**Reply by Geeyun-JY3** [reply]
https://greasyfork.org/reports/26522

> Seems to be allowed per MIT

忘记这事还是得看是否遵守 MIT License 了……

**Reply by the1812** [reply]
要说 MIT, 代码是可以随便 Fork 修改再发布, 但是我不受理来自这些再发布版本的反馈. 这也是我不在 GreasyFork 上发布的原因, 它 "安装在前 说明在后" 的设计很容易导致很多无效 issue.
既然管理觉得这个符合 MIT, 我就只好删掉一些反馈入口了.

**Comment by the1812** [comment]
是盗搬, 我从未授权在 GreasyFork 上发布

**Comment by Geeyun-JY3** [comment]
今天看这家伙的脚本都没了，也不知道后来发生了啥……
